### DISCUSSION TITLE
table "auth_indenties" column "name"

### DISCUSSION THREAD

**Discussion by jlopes90** [codeigniter4/shield#226] [Q&A] [answered]
column "name" what does it do?

**Comment by lonnie-vault** [comment, marked as answer]
I believe it's mostly unused in the current implementation, but it's there for other handlers to use. 

For example, if I was implementing a UI for the personal access tokens, I'd use the name to store a way for the user to recognize the PAT, like `Macbook Pro` or `Note 22+ Ultra` or whatever.
